cmd/getroughtime/main.go

```diff
@@ -62,19 +61,19 @@ func main() {
 		if err != nil {
 			logger.Fatal(err)
 		}
-		logger.Printf("delta: %v", delta.Truncate(time.Millisecond))
+		logger.Printf("Delta: %v", delta.Truncate(time.Millisecond))
 		os.Exit(0)
 	}
 
 	if *pingAddr != "" {
 		if *pingPubKey == "" {
-			logger.Fatal("ping: missing -pubkey")
+			logger.Fatal("Ping: missing -pubkey")
 		}
 		pk, err := base64.StdEncoding.DecodeString(*pingPubKey)
 		if err != nil {
-			logger.Fatalf("pubkey decode error: %s\n", err)
+			logger.Fatalf("Pubkey decode error: %s\n", err)
```

Comment:
Here and below:
```suggestion
			logger.Fatalf("Public key decode error: %s\n", err)
```

Reply:
My spelling linter just complained about starting a log output with a lower case letter, I didn't want to change the text but I can as well do that :+1: